pkg/kubectl/cmd/util/printing.go

```diff
@@ -30,6 +33,32 @@ func AddPrinterFlags(cmd *cobra.Command) {
 	cmd.Flags().StringP("template", "t", "", "Template string or path to template file to use when -o=template or -o=templatefile.  The template format is golang templates [http://golang.org/pkg/text/template/#pkg-overview]")
 }
 
+// AddOutputFlags adds output related flags to a command. Used by mutations only.
+func AddOutputFlags(cmd *cobra.Command) {
+	cmd.Flags().StringP("output", "o", "", "Output mode. Use -o name for showing only kind/name.")
```

Comment:
Using the same name for a flag that has different meanings depending on the command just doesn't sit right. I would prefer something like a boolean `-h` `--human-readable` and default it to true.

Or here's an idea: have all mutations do a `get` after successfully mutating, and then print the output for the `get`? Then all commands can take a `-o` flag, and we add `name` to the list of supported output formats, where `name` means just print `kind/resource`?